http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/BaseHttpAdvancedReactiveIT.java

```diff
@@ -182,6 +182,43 @@ public void multipartFormDataReader() {
                 .body(TEXT, is(TEXT));
     }
 
+    @Test
+    @DisplayName("RESTEasy Reactive Multipart Max Parameters test")
```

Comment:
The name of the method contains the same info, why do we need this?

Reply:
I added it to keep consistency with all the tests in the class that have got that annotation `@DisplayName`